internal/glfw/const.go

```diff
@@ -54,6 +54,8 @@ const (
 	MouseButtonLeft   = MouseButton(0)
 	MouseButtonRight  = MouseButton(1)
 	MouseButtonMiddle = MouseButton(2)
+	MouseButtonExtra1 = MouseButton(3)
+	MouseButtonExtra2 = MouseButton(4)
```

Comment:
This package is a package to emulate GLFW API, so the names here should be `MouseButton4` and `MouseButton5`.

https://pkg.go.dev/github.com/go-gl/glfw/v3.3/glfw#MouseButton